spec/takeaway_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway){ described_class.new(menu: menu, order: order, sms: sms, config: {}) }
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)